src/logging.h

```diff
@@ -59,6 +59,7 @@ namespace BCLog {
         VALIDATION  = (1 << 21),
         I2P         = (1 << 22),
         IPC         = (1 << 23),
+        RPCPARAMS   = (1 << 24),
```

Comment:
`test/functional/rpc_misc.py::L60` needs to be updated: `AssertionError: not(25 == 24)`